src/sentry/auth/superuser.py

```diff
@@ -275,12 +288,56 @@ def set_logged_in(self, user, current_datetime=None):
         request = self.request
         if current_datetime is None:
             current_datetime = timezone.now()
+
+        token = get_random_string(12)
+
+        if not self._needs_validation():
+            self._set_logged_in(
+                expires=current_datetime + MAX_AGE,
+                token=token,
+                user=user,
+                current_datetime=current_datetime,
+            )
+
+            logger.info(
+                "superuser.logged-in",
+                extra={"ip_address": request.META["REMOTE_ADDR"], "user_id": user.id},
+            )
+            return
+
+        try:
+            # need to use json loads as the data is no longer in request.data
+            su_access_json = json.loads(request.body)
```

Comment:
did you want to look into using request.POST instead?

Reply:
i looked into it, once the request reaches here, `request.POST` contains an empty query dict `<QueryDict: {}>`